### ISSUE TITLE
false positive: `authors' was removed in Emacs version 25.1. (emacs-lisp-package)

### ISSUE THREAD

**Issue by mooseyboots** [purcell/flycheck-package#12]
i'm receiving a false positive in flycheck, and i think it is due to flychck-package.

i let-bind `authors` somewhere, or use it as a function arg, and flycheck-package returns a flycheck error:

```
`authors' was removed in Emacs version 25.1. (emacs-lisp-package)
```

i'm not using it for anything built-in.

**Comment by purcell** [comment]
This is a `package-lint` issue really. And yeah, false positives are not too unusual — it's not practical to avoid all of them in `package-lint`, because doing so perfectly requires evaluating all the code being linted, so occasionally it can make sense to rename identifiers to avoid confusion.

Feel free to paste an example though.
